python-sdk/tutorials/automl-with-azureml/image-classification-multiclass-batch-scoring/auto-ml-image-classification-multiclass-batch-scoring.ipynb

```diff
@@ -922,9 +987,9 @@
    "tutorials"
   ],
   "kernelspec": {
-   "display_name": "Python 3.8 - AzureML",
+   "display_name": "Python [conda env:xaibenchmarks] *",
    "language": "python",
-   "name": "python38-azureml"
```

Comment:
I think this may need to stay like python38-azureml. I think it's a hint for CI users which built-in env to run the notebook on. May also want to revert other metadata changes in the notebook, like Python version, scrolled, etc.